policies/secpolicy.html

```diff
@@ -79,8 +101,8 @@ <h2>Issue severity</h2>
 	      <li>
 	      <em><a name="low">LOW</a> Severity.</em>
 	      This includes issues such as those that only affect the
-	      openssl command line utility, unlikely configurations, or hard
-	      to exploit timing (side channel) attacks.  These will in general
+	      openssl command line utility, or unlikely configurations
```

Comment:
missing a period, i think.
```suggestion
	      openssl command line utility, or unlikely configurations.
```